### PR TITLE
Add support for docker

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/glance/templates/default/glance-api.conf.erb [hunk 1]
```diff
@@ -81,6 +81,7 @@ workers = <%= [node["cpu"]["total"], 2, 4].sort[1] %>
 
 # Supported values for the 'container_format' image attribute
 #container_formats=ami,ari,aki,bare,ovf,ova
+container_formats=ami,ari,aki,bare,ovf,ova<%= @use_docker ? ",docker" : "" %>
```
Comment (tboerger) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: I think it should be nicer to have it as an attribute array and join it within the config, what do you think?

**Comment by tboerger** [comment]
Small nitpic but LGTM

**Comment by jsuchome** [comment]
+1, tested and adding docker image to glance works

File: chef/cookbooks/glance/templates/default/glance-api.conf.erb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: For the record, the reason to not use an attribute array is that we want to easily be able to compare the values we don't change (`ami,ari,aki,bare,ovf,ova`) with what is in the upstream config file.

**Pull request merged**